examples/simple/package.json

```diff
@@ -42,6 +42,7 @@
         "ra-input-rich-text": "^3.9.0",
         "ra-language-english": "^3.9.0",
         "ra-language-french": "^3.9.0",
+        "ra-ui-materialui": "^3.9.0",
```

Comment:
`ra-ui-materialui` is a dependency of `react-admin`, this shouldn't be necessary

Reply:
Yes that's what I thought as well. I'll try to update all the other dependencies then .